src/cart/utils/SConscript

```diff
@@ -1,32 +1,26 @@
-# (C) Copyright 2021-2022 Intel Corporation.
+# (C) Copyright 2021-2023 Intel Corporation.
 #
 # SPDX-License-Identifier: BSD-2-Clause-Patent
 #
 """Build crt_utils component"""
 
-LIB_SRC = ['crt_utils.c']
-
-
-####################################
-# Create cart utility share objects
-####################################
-def build_utility_shared_obj(env):
-    """Build cart utility library"""
-    cart_utils_objs = env.SharedObject(LIB_SRC, SHOBJPREFIX='s_')
-    Export('cart_utils_objs')
-
 
 def scons():
     """Scons function"""
 
-    Import('env', 'prereqs', 'conf_dir')
-
-    prereqs.require(env, 'protobufc')
+    Import('env', 'conf_dir')
 
     env.d_add_build_rpath()
 
+    cenv = env.Clone()
+
+    cenv.AppendUnique(CPPPATH=[Dir('../../mgmt').srcnode()])
+
+    cenv.require('protobufc', 'mercury')
```

Comment:
I think this change is what broke protobufc, the previous code was incorrectly modifying env so this was the right change but then protobufc should have been added back in src/test/ftest/cart.